crates/wysiwyg/src/composer_model/hyperlinks.rs

```diff
@@ -69,11 +69,15 @@ where
         link: S,
         text: S,
         suggestion: SuggestionPattern,
+        mut attributes: Vec<(S, S)>,
     ) -> ComposerUpdate<S> {
+        // when inserting a suggestion, ensure that it automatically has this attribute
+        attributes.push(("contenteditable".into(), "false".into()));
```

Comment:
Ok. I'll close this PR down and open up a new one (just because there have been a few different strands of work for me which are all similar but not identical) where the clients are responsible for passing in `contenteditable`. I'll use the work that David did to try and come up with some swift code